docs/dev-guide/contribute/commit_the_code_and_submit_a_pull_request.md

```diff
@@ -42,10 +42,18 @@ git push -u origin my-branch
 
 ## Submit a pull request
 Once committed your change, you can go back to TiSpark and create a pull request by clicking one of the buttons in the following picture.
-![pr](pics/pr.png)
+![pr](../pics/pr.png)
 We have a template for pull request . Please follow the guide and fill it.
-![pr template](pics/pr_templete.png)
+![pr template](../pics/pr_templete.png)
 
 ## Conflict
 There may be a conflict when you submit a pull request since others may change the same code as yours. Git can't handle it automatically, you need to do it yourself. If you are not sure about how to handle it, feel free to ask reviewers for help.
-![conflict](pics/conflict.png)
+![conflict](../pics/conflict.png)
+
+## Review
+
+Ask the reviewer to review your code. If there is a problem, you need to modify it and push it again. The reviewer will review it again until it is approved.
```

Comment:
Sometimes, it is not a problem, just a comment or advice. 
How about changing it to "If the reviewers leave some comments, you should resolve them by modifying the code or explaining the reasons." ?